packages/hdwallet-core/src/ethereum.ts

```diff
@@ -12,6 +12,9 @@ export interface SerializedEthereumRpcError {
   stack?: string;
 }
 
+type Address = `0x${string}`;
+type Hex = `0x${string}`;
```

Comment:
probably worth exporting if needed.

Reply:
isn't for the time being in hdwallet-core, so probably not worth it?

Reply:
more from a package consumption point of view. not a huge deal since we will be using the comparable viem type in web